job-server-extras/src/test/scala/spark/jobserver/python/PythonJobManagerSpec.scala

```diff
@@ -22,29 +22,30 @@ class PythonJobManagerSpec extends ExtrasJobSpecBase(PythonJobManagerSpec.getNew
     it("should work with JobManagerActor") {
       val pyContextConfig = ConfigFactory.parseString(
         """
-          |context-factory = "spark.jobserver.python.PythonSparkContextFactory"
+          |context-factory = "spark.jobserver.python.PythonSessionContextFactory"
           |context.name = "python_ctx"
           |context.actorName = "python_ctx"
         """.stripMargin).
         withFallback(PythonSparkContextFactorySpec.config)
       manager = system.actorOf(JobManagerActor.props(pyContextConfig, daoActor))
 
       manager ! JobManagerActor.Initialize(None)
-      expectMsgClass(classOf[JobManagerActor.Initialized])
+      expectMsgClass(30 seconds, classOf[JobManagerActor.Initialized])
 
       uploadTestEgg("python-demo")
 
       manager ! JobManagerActor.StartJob(
         "python-demo",
-        "example_jobs.word_count.WordCountSparkJob",
+        "example_jobs.word_count.WordCountSparkSessionJob",
         ConfigFactory.parseString("""input.strings = ["a", "b", "a"]"""),
         errorEvents ++ syncEvents)
-      expectMsgPF(3 seconds, "Expected a JobResult or JobErroredOut message!") {
+      expectMsgPF(15 seconds, "Expected a JobResult or JobErroredOut message!") {
```

Comment:
I was getting a timeout error on this but testing it with 3 seconds now I can no longer reproduce. Reverted the change